xwiki-pro-macros-api/src/main/resources/css/userlist.css

```diff
@@ -1,3 +1,14 @@
-.xwiki-userlist-user-avatar {
-  width: 50px;
+.xwiki-userlist-fixed-layout {
+  table-layout: fixed;
+}
+
+/** Make sure table cells do not overflow when containing for instance long e-mail addresses. */
+.xwiki-userlist tr td {
```

Comment:
A ``td`` is always inside a ``tr`` so I'm not sure what's the benefit of specifying ``tr`` in the selector. Same below.

Reply:
Right, I have removed the `tr` in both locations.